src/main/java/mate/academy/controllers/CinemaHallController.java

```diff
@@ -0,0 +1,36 @@
+package mate.academy.controllers;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import mate.academy.model.dto.request.CinemaHallRequestDto;
+import mate.academy.model.dto.response.CinemaHallResponseDto;
+import mate.academy.service.CinemaHallService;
+import mate.academy.service.mapper.CinemaHallMapper;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController("/cinema-halls")
```

Comment:
does it work? please attach the screenshot from Postman/browser where you fetch all cinema-halls

Reply:
![image](https://user-images.githubusercontent.com/48122349/97329113-61767b00-187f-11eb-88f9-a18e4de6f33e.png)